package.json

```diff
@@ -18,12 +18,12 @@
     "@hapi/boom": "9.x.x",
     "@hapi/bounce": "2.x.x",
     "@hapi/hoek": "9.x.x",
-    "@hapi/joi": "17.x.x"
+    "@hapi/validate": "1.x.x"
```

Comment:
I think the source code will need to be updated as well.

Reply:
Yep sorry. I was expecting it to complain but I guess it still had joi from another dependency.